plugins/system/shortcut/shortcut.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="plugin" group="system" method="upgrade">
+	<name>plg_system_shortcut</name>
+	<author>Joomla! Project</author>
+	<creationDate>Jun 2022</creationDate>
+	<copyright>(C) 2022 Open Source Matters, Inc.</copyright>
+	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
+	<authorEmail>admin@joomla.org</authorEmail>
+	<authorUrl>www.joomla.org</authorUrl>
+	<version>__DEPLOY_VERSION__</version>
+	<description>PLG_SYSTEM_SHORTCUT_XML_DESCRIPTION</description>
+	<namespace path="src">Joomla\Plugin\System\Shortcut</namespace>
+	<media destination="plg_system_shortcut" folder="media">
+	  <folder>js</folder>
+	</media>
+	<files>
+		<folder>fields</folder>
+		<filename plugin="shortcut">shortcut.php</filename>
+	</files>
+	<languages>
+		<language tag="en-GB">language/en-GB/plg_system_shortcut.ini</language>
+		<language tag="en-GB">language/en-GB/plg_system_shortcut.sys.ini</language>
+	</languages>
+	<config>
+		<fields name="params">
+			<fieldset
+				name="core"
+				label="PLG_SYSTEM_SHORTCUT_CORE_SHORTCUTS_TAB"
+				addfieldprefix="Joomla\Plugin\System\Shortcut\Field"
+				>
+				<fieldset
+					name="generic_buttons"
+					label="PLG_SYSTEM_SHORTCUT_GENERIC_BUTTONS_FIELDSET_LABEL"
+					>
+					<field
+						name="helpKey"
+						type="Keyselect"
+						label="PLG_SYSTEM_SHORTCUT_BUTTON_HELP_LBL"
+						default="ALT + X"
+					/>
+					<field
+						name="searchKey"
+						type="Keyselect"
+						label="PLG_SYSTEM_SHORTCUT_SEARCH_LBL"
+						default="S"
```

Comment:
reposting as my comment was lost when the code was moved.

Using a single character without a modifier is a really bad idea. it should be obvious why

Reply:
and in case it is not. try and hit the S hotkey when you are focussed on the sort filter

Reply:
I think I fixed it. When I have focus on the checkbox or the column filter I can hit J+S and jump into the search input now